frontend/src/views/SearchResults/ReportIssue.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="grid grid-cols-12">
+    <div class="col-span-12 md:col-span-7 md:pr-10">
+      <h4>Une erreur ? Signalez-là ici</h4>
+      <p class="m-0">
+        Aidez-nous à améliorer la qualité de nos données, en remontant toute erreur ou incohérence que vous pourriez
+        constater. Nous vous remercions d'avance. 🙏🏼
```

Comment:
```suggestion
        constater. Nous vous remercions d'avance 🙏🏼
```
Je suis tombé dans une spirale sur l'usage des emojis à coté de signes de ponctuation, il paraît [qu'ils les substituent](https://www.radiofrance.fr/franceculture/une-histoire-de-la-ponctuation-l-emoji-point-du-futur-5397764) :laughing: